[gatsby-plugin-theme-ui] theme-ui@^0.1.2 peerDeps warning

Here is my NPM warning
```
npm WARN gatsby-plugin-theme-ui@0.2.25 requires a peer of theme-ui@^0.1.2 but none is installed. You must install peer dependencies yourself.
```

## Related code

https://github.com/system-ui/theme-ui/blob/5e4642258026ecc2ac6c61348b2f635435f5abc9/packages/gatsby-plugin-theme-ui/package.json#L8

@jxnblk Yes.
Here are my `npm ls theme-ui` output:
```
`-- UNMET PEER DEPENDENCY theme-ui@0.2.25

npm ERR! peer dep missing: theme-ui@^0.1.2, required by gatsby-plugin-theme-ui@0.2.25
```

Some part of my deps:
```json
"dependencies": {
    "@emotion/core": "^10.0.14",
    "@mdx-js/react": "^1.1.4",
    "gatsby": "^2.13.50",
    "gatsby-plugin-theme-ui": "^0.2.25",
    "react": "^16.8.6",
    "react-dom": "^16.8.6",
    "theme-ui": "^0.2.25",
  },
```

--- a/packages/gatsby-plugin-theme-ui/package.json
+++ b/packages/gatsby-plugin-theme-ui/package.json
@@ -5,7 +5,7 @@
   "author": "Brent Jackson",
   "license": "MIT",
   "peerDependencies": {
-    "theme-ui": "^0.1.2"
+    "theme-ui": "^0.2.0"
   },
   "keywords": [
     "gatsby",

